tests/trx_generator/trx_generator.hpp

```diff
@@ -15,8 +18,66 @@ namespace eosio::testing {
       fc::crypto::private_key _signer;
    };
 
+   struct action_pair_w_keys {
+      action_pair_w_keys(eosio::chain::action first_action, eosio::chain::action second_action, fc::crypto::private_key first_act_signer, fc::crypto::private_key second_act_signer)
+            : _first_act(first_action), _second_act(second_action), _first_act_priv_key(first_act_signer), _second_act_priv_key(second_act_signer) {}
+
+      eosio::chain::action _first_act;
+      eosio::chain::action _second_act;
+      fc::crypto::private_key _first_act_priv_key;
+      fc::crypto::private_key _second_act_priv_key;
+   };
+
+   struct account_name_generator {
+      account_name_generator() : _name_index_vec(ACCT_NAME_LEN, 0) {}
+
+      const char* CHARMAP = "12345abcdefghijklmnopqrstuvwxyz";
+      const int ACCT_NAME_CHAR_CNT = 31;
+      const int ACCT_NAME_LEN = 12;
+      const int MAX_PREFEX = 960;
+      std::vector<int> _name_index_vec;
+
+      void increment(int index) {
+         _name_index_vec[index]++;
+         if(_name_index_vec[index] >= ACCT_NAME_CHAR_CNT) {
+            _name_index_vec[index] = 0;
+            increment(index - 1);
+         }
+      }
+
+      void increment() {
+         increment(_name_index_vec.size() - 1);
+      }
+
+      void incrementPrefix() {
+         increment(1);
+      }
+
+      void setPrefix(int id) {
+         if (id > MAX_PREFEX) {
+            elog("Account Name Generator Prefix above allowable ${max}", ("max", MAX_PREFEX));
+            return;
+         }
+         _name_index_vec[0] = 0;
+         _name_index_vec[1] = 0;
+         for(int i = 0; i < id; i++) {
+            incrementPrefix();
+         }
+      };
+
+      std::string calcName() {
+         std::string name;
+         name.reserve(12);
+         for(auto i: _name_index_vec) {
+            name += CHARMAP[i];
+         }
+         return name;
+      }
+   };
+
    struct trx_generator_base {
       p2p_trx_provider _provider;
+      uint16_t _id;
```

Comment:
Init to `0` here. I realize it is done in the constructor, but safer to init here just in case.

Reply:
Addressed: 1ad0888997